pallets/oracle/src/oracle.rs

```diff
@@ -329,6 +334,43 @@ pub mod tests {
         Ok(())
     }
 
+    #[test]
```

Comment:
would love to see another test for `parsed.timestamp == 0` i think it is a conceivable "default value" edge case on the exchange side